corehq/apps/callcenter/tests/test_indicators.py

```diff
@@ -637,3 +636,8 @@ def test_standard_indicators(self, mock):
                 custom_cache=locmem_cache
             )
             self._test_indicators(self.cc_user, indicator_set.get_data(), expected_standard_indicators())
+
+
+@use_sql_backend
+class TestSavingToUCRDatabaseSQL(TestSavingToUCRDatabase):
+    pass
```

Comment:
👍 
That took me by surprise when all tests ran for the class inherited from, while i was working for some tests. Great use of it here.